spec/database_connection_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'database_connection'
+
```

Comment:
this is minor but the require expressions from the spec files could be moved to the spec_helper file